cmd/ctr/app/main.go

```diff
@@ -27,6 +27,7 @@ import (
 	"github.com/containerd/containerd/cmd/ctr/commands/install"
 	"github.com/containerd/containerd/cmd/ctr/commands/leases"
 	namespacesCmd "github.com/containerd/containerd/cmd/ctr/commands/namespaces"
+	ociCmd "github.com/containerd/containerd/cmd/ctr/commands/oci"
```

Comment:
ocicmd is more go-style import name😄

Reply:
there are `namespacesCmd` and `versionCmd`, so I followed the same pattern.